website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -32,10 +33,10 @@ An API token contains the following pieces of information:
 
 | Name | Description |
 | --- | --- |
-| Token name (sometimes called "username") | The token's name. Names or **not** required to be unique. |
-| Type | What kind of token it is: admin, client, or front-end. |
-| Projects | What projects a token has access to. |
-| Environment | What environment the token has access to. |
+| Token name (sometimes called "username") | The token's name. Names are **not** required to be unique. |
+| Type | What kind of token it is: admin, client, user (personal api token) or front-end. |
+| Projects | What projects a token has access to - not present in personal api tokens. |
+| Environment | What environment the token has access to - not present in personal api tokens. |
```

Comment:
I wonder if personal api tokens should have a different table: they don't have projects or environments, but they do have an additional expiration date that the others do not. Thoughts?

Reply:
Ye, that's a pretty good idea actually, let me see what I can do here

Reply:
...actually on second thought, I'm not sure that PATs want their own table - they don't actually include any of the information described in the table. Maybe just a note explaining their format? WDYT?

Reply:
Interesting. Do personal API tokens have names? I know that GitHub lets you name them (so you can have different ones for different purposes). But I think you're going in the right direction. In that case, we should probably point out that the table only applies to the other API tokens and ask the user to refer to the PAT section for info on them.

Reply:
They have a description but not a name, serves the same purpose as GH I think.

I've just added some text, but I haven't linked it to anything - we could link to the section higher up but that feels redundant, or the how to guide but I'm not sure that makes life better. Will you give it some eyeballs and see how you feel?